addons/sale/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,197 @@
+---Fields in module 'sale'---
+sale         / account.analytic.line    / so_line (many2one)            : NEW relation: sale.order.line
```

Comment:
This is used to create sale orders for services or expenses. I don't think that functionality exists in Odoo 8.0.